bin/zerobufCxx.py

```diff
@@ -15,6 +15,8 @@
 import sys
 from collections import namedtuple, OrderedDict
 
+inline_implementation = False
```

Comment:
Unless there is a good reason for this global "option" I would remove it and always add inline